src/payload.rs

```diff
@@ -248,9 +250,10 @@ impl Payload {
 
             file.write_all_at(&decoded, dst_extent.start_block() * BLOCK_SIZE)?;
 
-            progress_bar.inc(decoded.len() as u64);
+            if !quiet {
+                progress_bar.inc(decoded.len() as u64);
+            }
```

Comment:
Instead of this. use `with_draw_target(ProgressDrawTarget::hidden())` on the `progress_bar`